vue-components/tests/e2e/specs/Lookup.test.js

```diff
@@ -1,50 +1,44 @@
-module.exports = {
-	'Lookup Menu shows on user input': ( client ) => {
+describe( 'Lookup Menu', function () {
```

Comment:
Why are we not using arrow functions, i.e., `() => { ... }`, here?

Reply:
Since the docs specify the ability to change general testing settings with `this` I decided to just follow their example and use functions with a `this`context. See https://nightwatchjs.org/guide/using-nightwatch/using-bdd-describe.html